replayserver/streams/delayed.py

```diff
@@ -66,17 +66,41 @@ def _data_length(self):
         return min(len(self._stream.data), self._current_position)
 
     def _data_slice(self, s):
+        if isinstance(s, slice):
+            return self._get_slice(s)
```

Comment:
if s is instance of global method method `slice` https://www.programiz.com/python-programming/methods/built-in/slice
call `_get_slice` with `s`, where on function `s` will be variable `stop`...

Reply:
`slice is type(slice(1,1,1)) == True`

Reply:
??

Reply:
`slice` is a class, not a function

Reply:
Classes in python should be `CamelCase`

Reply:
When `slice` is internal python function. from __builtins__ import slice

Reply:
Straight from the interpreter:
```
>>> slice
<class 'slice'>
```